site/shared/src/main/scala/shared/User.scala

```diff
@@ -23,3 +23,26 @@ case class UserProgress(
   args:         Option[String],
   succeeded:    Boolean
 )
+
+case class OverallUserProgressItem(
+  libraryName: String,
+  sections:    Int,
+  completed:   Int
+)
```

Comment:
If you can provide also the color of the library, we'll be able to use the service to draw the list of libraries in the home screen, with the user progress. 
Could you do it please?